src/vfio_device.rs

```diff
@@ -48,6 +54,8 @@ pub enum VfioError {
     GroupGetDeviceFD,
     #[cfg(feature = "kvm")]
     KvmSetDeviceAttr(SysError),
+    #[cfg(feature = "mshv")]
+    MshvSetDeviceAttr(SysError),
```

Comment:
Could we merge "KvmSetDeviceAttr" and "MshvSetDeviceAttr" as "SetDeviceAttr"? It may help to simplify the clients.

Reply:
Yes we can. I just didn't want to break existing clients. I can rename `KvmSetDeviceAttr` to `SetDeviceAttr` if people are okay with suddenly losing a variant and hence breaking existing users of this library.

Reply:
I've changed `KvmSetDeviceAttr` to `SetDeviceAttr` in the latest patch.